webpack.config.js

```diff
@@ -12,16 +12,6 @@ const isProduction = process.env.NODE_ENV == 'production';
 
 const stylesHandler = MiniCssExtractPlugin.loader;
 
-function resolve (dir) {
```

Comment:
I think this should not be removed (it's causing an error)

Reply:
something got mixed up. Sorry. I have to clean it.